index.html

```diff
@@ -9,53 +9,100 @@
-                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Enabled.svg"></a></li>
-                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Vector.svg"></a></li>
-                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Disabled.svg"></a></li>
-          </ul>
+                
+                                <ul class="social_icon">
+                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/gi.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/in.svg"></a> </li>
+                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Enabled.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Vector.svg"></a></li>
+                                        <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Disabled.svg"></a></li>
+                                  </ul>
+                 
+
+                    <div class="grid-container">
+                       <div class="grid-item texts">
+                                    
+                                        <h1>Hey There. I’m Alex</h1>
+                                        <span>I’am a Software Developer</span>
+                                        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                                    
+                            </div>
+                            <ul class="social_iconx">
+                                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/gi.svg"></a></li>
+                                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/in.svg"></a> </li>
+                                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Enabled.svg"></a></li>
+                                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Vector.svg"></a></li>
+                                <li><a href="#"><img class="icon" alt="social_icon" src="imgs/Disabled.svg"></a></li>
+                          </ul>
         </div>
+    </div>
     </header>
 
     
     <section class="container">
         <div class="works">
-            <div class="works-title">
-                <h1>My Recent Works</h1>
-                <div class="line">
-                </div>
-            </div>
+            <article class="featured2">
+                    <div class="works-title">
+                       <h1>My Recent Works</h1>
+                    </div>
+                      <div class="multi-post">
+                        <div class="line"></div>
+                      </div></article>
+          </div>
+        
+
+            <article class="featured">
```

Comment:
- [ ] I kindly suggest that you add your `see project` button that takes you to the Live Demo of the featured project or to the GitHub repository.

| Your Design  | Template Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/178112208-d8813be9-7aef-4d73-8023-77a7bb93911d.png) |  ![image](https://user-images.githubusercontent.com/52934128/178112159-ce45e9d9-9a8d-4c58-bf5c-07f08fa109ba.png)|